README.md

```diff
@@ -55,14 +54,30 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
 
 ## Available Commands <a name="commands"></a>
 
-The following table lists the available commands for the plugin and their explanation:
+The following section lists the available commands for the plugin and their explanation:
```

Comment:
Probably you would like to update this section (Available commands) like this one https://gist.github.com/budhrg/1321e995492a86c0df289376e9666531#available-commands.

If you want, just edit this "gist", and add the changes. I have updated this gist with required changes for unnecessary brackets